### PR TITLE
repoquery: Add a test for disabling modular filtering

### PR DESCRIPTION
**Pull request by lukash**

### CODE REVIEW

File: dnf-behave-tests/features/repoquery/modules.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Feature: repoquery tests for handling modularity.
+
+Background:
+ Given I use repository "repoquery-modules"
+
+
+Scenario: Test --disable-modular-excludes
```
Comment (pkratoch) [inline dnf-behave-tests/features/repoquery/modules.feature]: Can you please fix the option name here? Should be ``--disable-modular-filtering``.

**Comment by pkratoch** [comment]
Can you also add one non-modular nodejs package? It should also be visible with ``--disable-modular-filtering`` only.

Also, how about adding one scenario for enabled stream, too?

**Comment by lukash** [comment]
@pkratoch I've fixed the scenario description, added a non-modular package and a test for an enabled module stream.

**Comment by pkratoch** [comment]
Thanks for the changes. Looks good to me now; I will merge it once the dnf PR is ready.

**Pull request merged**